src/App.tsx

```diff
@@ -17,18 +46,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                currentStatus={filterStatus}
+                onFilterChange={setFilterStatus}
+                query={query}
+                onQueryChange={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isLoading && todos.length > 0 ? (
+                <TodoList
+                  todos={getFilteredTodos()}
```

Comment:
It seems like `getFilteredTodos` is being used correctly here as a function to filter todos. However, ensure that it is defined and used consistently as a function throughout your application to avoid any potential issues.